mods/hide-dotfiles-explorer.wh.cpp

```diff
@@ -0,0 +1,311 @@
+// ==WindhawkMod==
+// @id              hide-dotfiles-explorer
+// @name            Hide Dotfiles (Explorer Only)
+// @description     Hide dotfiles (files starting with .) in Windows Explorer only. Configure excluded paths in settings.
+// @version         1.0
+// @author          @danalec
+// @github          https://github.com/danalec
+// @include         explorer.exe
+// @compilerOptions -lcomctl32 -lole32 -loleaut32 -lshell32 -lshlwapi
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+This mod automatically hides files and folders that start with a dot (.) in Windows Explorer, similar to Unix/Linux systems.
+
+## Features
+- Hides all files/folders starting with "." (like .git, .vscode, .env, etc.)
+- Works only in Windows Explorer (doesn't affect file dialogs or other applications)
+- Configurable exclusion paths where dotfiles will remain visible
+
+## Configuration
+You can exclude specific paths, folders, or files from having their dotfiles hidden. In the mod settings, add paths separated by commas:
+- Single path: C:\Projects
+- Multiple paths: C:\Program Files", "D:\My Documents"
+
+You can also exclude specific folders or files directly, for example:
+- Folder: C:\Projects\.git
+- File: C:\Projects\.env
```

Comment:
So, can you just specify ".git" or ".env" to exclude all such folders/files? If yes, I think it's worth documenting it.